plugins/continuous-toolbox/src/RecyclableBlockFlyoutInflater.ts

```diff
@@ -0,0 +1,193 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * A block inflater that caches and reuses blocks to improve performance.
+ */
+export class RecyclableBlockFlyoutInflater extends Blockly.BlockFlyoutInflater {
+  /**
+   * Whether or not block recycling is enabled.
+   */
+  private recyclingEnabled = false;
```

Comment:
Similarly to previously: Can this not be `public`, obviating the need for `setRecyclingEnabled`?